apps/accounts/views.py

```diff
@@ -71,7 +71,22 @@ def inner(self, request):
                                            config=settings.FXA_CONFIG)
         except verify.IdentificationError:
             return Response({'error': 'Profile not found.'}, status=401)
-        return fn(self, request, user=find_user(identity), identity=identity)
+        else:
+            identity_user = find_user(identity)
+            if request.user.is_authenticated():
```

Comment:
my concern was that some refactoring might cause a view to call `login_user()` without first checking that `request.user` matches `user`. It's low level enough though that hopefully someone wouldn't do that. Maybe an underscore would make me feel better, jk ;) No biggie.